package.json

```diff
@@ -66,6 +67,8 @@
     "aws-lambda-mock-context": "^3.2.1",
     "chai": "^4.2.0",
     "gulp": "^4.0.0",
+    "jest": "^24.9.0",
+    "jest-plugin-context": "^2.9.0",
```

Comment:
will also need packages for sonar see details in  [defect commit](https://github.com/dvsa/cvs-svc-defects/commit/42bddbc8803d0965d950cc8fdb5bc694db16bd70)